ddtrace/internal/telemetry/__init__.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from ...internal.utils.formats import asbool
+from ...internal.utils.formats import get_env
+from .data.integration import Integration
+from .data.payload import AppStartedPayload
+from .data.telemetry_request import TelemetryRequest
+from .data.telemetry_request import create_telemetry_request
+from .telemetry_writer import TelemetryWriter
+
+
+_TELEMETRY_WRITER = None  # type: Optional[TelemetryWriter]
+
+TELEMETRY_ENABLED = asbool(get_env("instrumentation_telemetry", "enabled", default=False))  # type: bool
+
+if TELEMETRY_ENABLED:
+    _TELEMETRY_WRITER = TelemetryWriter()
```

Comment:
do we need to explicitly start the writer?

if so we could always create the writer, but only start it if it is enabled.

then we can have the queuing methods on the writer be a no-op if it is not enabled (or running?)